integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/viewpopenqueueservice/test/ViewPopEnqueueTest.java

```diff
@@ -0,0 +1,120 @@
+package org.wso2.carbon.esb.viewpopenqueueservice.test;
```

Comment:
Need to add the licence header.

Reply:
This test has been removed from the PR and no longer exists